common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -57,279 +60,175 @@ nav.sequence-nav {
-          background: #333;
-          color: $white;
-          font-family: $sans-serif;
-          line-height: lh();
-          right: 0;
-          opacity: 0.0;
-          padding: 6px;
-          position: absolute;
-          top: 48px;
-          text-shadow: 0 -1px 0 $black;
-          @include transition(all .1s $ease-in-out-quart 0s);
-          white-space: pre;
-          visibility: hidden;
-          pointer-events: none;
-
-
-          &:empty {
-            background: none;
-
-            &::after {
-              display: none;
+              &.active {
+                @extend .active;
+                background-image: url('../images/sequence-nav/video-icon-current.png');
+              }
             }
-          }
 
-          &::after {
-            background: #333;
-            content: " ";
-            display: block;
-            height: 10px;
-            right: 18px;
-            position: absolute;
-            top: -5px;
-            @include transform(rotate(45deg));
-            width: 10px;
-          }
-        }
-
-        &:hover, &:focus {
-          p {
-            display: block;
-            margin-top: ($baseline/5);
-            opacity: 1.0;
-            visibility: visible;
-          }
-        }
-      }
-    }
-  }
+            //other
+            &.seq_other {
+              &.inactive {
+                background-image: url('../images/sequence-nav/document-icon-normal.png');
+              }
 
-  ul {
-    position: absolute;
-    top: 0;
-    list-style: none !important;
-    height: 100%;
-    right: 0;
-    top: 0;
-    width: 100%;
-    margin: 0;
-    border: none;
-
-    li {
-      position: absolute;
-      margin-bottom: 0;
-      height: 44px;
-      width: 70px;
-      border: 1px solid $gray-l3;
-      @include linear-gradient(top, #eee, #ddd);
-      box-shadow: 0 1px 0 rgba(255, 255, 255, .7) inset;
-      z-index: 1;
-
-      &.prev, &.next {
-
-        a {
-          background-position: center;
-          background-repeat: no-repeat;
-          display: block;
-          height: 100%;
-          width: 40px;
-          text-indent: -9999px;
-          overflow: hidden;
-          @include transition(all .2s $ease-in-out-quad 0s);
-
-/*          &:focus {
-            outline: 0;
-          }
-*/
-          &:hover, &:focus {
-            opacity: 0.5;
-          }
+              &.visited {
+                background-image: url('../images/sequence-nav/document-icon-visited.png');
+              }
 
-          &.disabled {
-            cursor: normal;
-            opacity: 0.4;
-          }
-        }
-      }
+              &.active {
+                background-image: url('../images/sequence-nav/document-icon-current.png');
+              }
+            }
 
-      &.prev {
-        @include left(-10px);
-        @include border-radius(35px, 0, 0, 35px);
+            //vertical & problems
+            &.seq_vertical, &.seq_problem {
+              &.inactive {
```

Comment:
NIT: I know you're just moving style rule order here in your diff, but any place we can change to use the SMACSS stateful classes (like here), I'm all for. This may not be the time (due to more logical/test dependencies), but if its opportunistic, go for it.